### PR TITLE
docs(controllers): add warning about route parameter ordering

### PR DESCRIPTION
**Pull request by tgallacher**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

Docs fail to call out important fact regarding route definition order, when defining route parameters. 

## What is the new behavior?

The importance of route parameter order is now called out. 

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: content/controllers.md [hunk 1]
```diff
@@ -268,6 +268,8 @@ getDocs(@Query('version') version) {
 
 Routes with static paths won't work when you need to accept **dynamic data** as part of the request (e.g., `GET /cats/1` to get cat with id `1`). In order to define routes with parameters, we can add route parameter **tokens** in the path of the route to capture the dynamic value at that position in the request URL. The route parameter token in the `@Get()` decorator example below demonstrates this usage. Route parameters declared in this way can be accessed using the `@Param()` decorator, which should be added to the method signature.
 
+> info **Warning** Routes with parameters should be defined after any static routes. This prevents the parameterized routes from intercepting traffic destined for the static routes.
```
Comment (micalevisk) [inline content/controllers.md:271]:
```suggestion
> info **Warning** Routes with parameters should be defined after any static paths. This prevents the parameterized routes from intercepting traffic destined for the static paths.
```

as this page uses 'static paths' everywhere, I think keeping the same word sounds less confusing

**Review by micalevisk — APPROVED** [review]

**Comment by tgallacher** [comment]
@micalevisk what's the process to get this merged 🙏

**Comment by micalevisk** [comment]
@tgallacher Kamil will review this as soon as he can, don't worry :) 

thanks

File: content/controllers.md [hunk 2]
```diff
@@ -268,6 +268,8 @@ getDocs(@Query('version') version) {
 
 Routes with static paths won't work when you need to accept **dynamic data** as part of the request (e.g., `GET /cats/1` to get cat with id `1`). In order to define routes with parameters, we can add route parameter **tokens** in the path of the route to capture the dynamic value at that position in the request URL. The route parameter token in the `@Get()` decorator example below demonstrates this usage. Route parameters declared in this way can be accessed using the `@Param()` decorator, which should be added to the method signature.
 
+> info **Warning** Routes with parameters should be declared after any static paths. This prevents the parameterized paths from intercepting traffic destined for the static paths.
```
Comment (kamilmysliwiec) [inline content/controllers.md:271]:
```suggestion
> info **Hint** Routes with parameters should be declared after any static paths. This prevents the parameterized paths from intercepting traffic destined for the static paths.
```

**Comment by kamilmysliwiec** [comment]
lgtm

**Pull request merged**